README.md

```diff
@@ -176,18 +176,29 @@ search for a Python interpreter matching that version in the following order:
 
 ### Installing into arbitrary Python environments
 
-Since uv has no dependency on Python, it can even install into virtual environments other than
+Since uv has no dependency on Python, it can install into virtual environments other than
 its own. For example, setting `VIRTUAL_ENV=/path/to/venv` will cause uv to install into
-`/path/to/venv`, no matter where uv is installed.
-
-uv can also install into arbitrary, even non-virtual environments by providing a `--python` argument
-to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python` will
-install into the environment linked to the `/path/to/python` interpreter.
-
-For convenience, `uv pip install --system` will install into the system Python environment, as an
-approximate shorthand for, e.g., `uv pip install --python=$(which python3)`. Though we generally
-recommend the use of virtual environments for dependency management, `--system` is intended to
-enable the use of uv in continuous integration and containerized environments.
+`/path/to/venv`, regardless of where uv is installed. Note that if `VIRTUAL_ENV` is set to
+a directory that is **not** a [PEP 405 compliant](https://peps.python.org/pep-0405/#specification)
+virtual environment, it will be ignored.
+
+uv can also install into arbitrary, even non-virtual environments, with the `--python` argument
+provided to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python`
+will install into the environment linked to the `/path/to/python` interpreter.
+
+For convenience, `uv pip install --system` will install into the system Python environment.
+Using `--system` is roughly equivalent to `uv pip install --python=$(which python)`,
+but note it will skip `python` executables on the `PATH` that execute in a virtual environment.
```

Comment:
Nit: "but note that"

Reply:
Could "executables on the `PATH` that execute in a virtual environment." be something like... "executables that are linked to virtual environments"?

Reply:
```suggestion
but note that executables that are linked to virtual environments will be skipped.
```